urllib3/util/selectors.py

```diff
@@ -510,7 +512,7 @@ def select(self, timeout=None):
             ready_fds = {}
 
             kevent_list = _syscall_wrapper(self._kqueue.control, True,
```

Comment:
So we're going to have to make a wrapper for `select.kqueue.control` as it [doesn't take key-word arguments](https://travis-ci.org/shazow/urllib3/jobs/244134449#L1750) and `timeout` is always passed in as a key-word argument in the `_syscall_wrapper`. Look at the `SelectSelector` for an example of this.